database/db.js

```diff
@@ -12,7 +12,7 @@ let grabTimeSlots = (rest_id, date, cb) => {
     if (error) {
       throw error;
     }
-    cb(results);
+    cb(results); 
```

Comment:
Nitpicky, but perhaps give your callback a more descriptive name to distinguish it from several callbacks in your code?